app/components/team_list/team_list_item/team_list_item.tsx

```diff
@@ -68,17 +69,19 @@ export default function TeamListItem({team, textColor, iconTextColor, iconBackgr
             type='opacity'
             style={styles.touchable}
         >
-            <View style={styles.icon_container}>
-                <TeamIcon
-                    id={team.id}
-                    displayName={displayName}
-                    lastIconUpdate={lastTeamIconUpdateAt}
-                    selected={false}
-                    textColor={iconTextColor || theme.centerChannelColor}
-                    backgroundColor={iconBackgroundColor || changeOpacity(theme.centerChannelColor, 0.16)}
-                    testID={`${teamListItemTestId}.team_icon`}
-                />
-            </View>
+            {!hideIcon &&
+                <View style={styles.icon_container}>
+                    <TeamIcon
+                        id={team.id}
+                        displayName={displayName}
+                        lastIconUpdate={lastTeamIconUpdateAt}
+                        selected={false}
+                        textColor={iconTextColor || theme.centerChannelColor}
+                        backgroundColor={iconBackgroundColor || changeOpacity(theme.centerChannelColor, 0.16)}
+                        testID={`${teamListItemTestId}.team_icon`}
+                    />
+                </View>
+            }
             <Text
                 style={[styles.text, Boolean(textColor) && {color: textColor}]}
                 numberOfLines={1}
```

Comment:
for long names, should we add an ellipsis ?

Reply:
It's already handled! 
![image](https://github.com/user-attachments/assets/7b0da15a-422b-4399-9e08-286951bb108c)